courses/machine_learning/deepdive2/feature_engineering/solutions/3_Basic_Feature_Engineering_in_Keras.ipynb

```diff
@@ -0,0 +1,2565 @@
+{
```

Comment:
<p>instead of this cell and the one below, I'd just do </p><p>housing_df.describe()</p><p>and then interpret the 'count' value to see nothing is missing. You also get other descriptive statistics with .describe()</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>